hack/make-apimachinery.sh

```diff
@@ -30,16 +30,15 @@ mkdir -p "${WORK_DIR}/go/"
 cp -R "${GOPATH}/src/k8s.io/kops/vendor/" "${WORK_DIR}/go/src"
 
 unset GOBIN
-unset GOFLAGS
 
-env GOBIN="${WORK_DIR}/go/bin" GOPATH="${WORK_DIR}/go/" go install -v k8s.io/code-generator/cmd/conversion-gen/
+env GOBIN="${WORK_DIR}/go/bin" GOPATH="${WORK_DIR}/go/" go install -mod=mod -v k8s.io/code-generator/cmd/conversion-gen/
```

Comment:
Not sure whether this should be mod or vendor?  But I don't think it's a blocker.

Reply:
We're not vendoring k8s.io/code-generator so I believe this was always installing from upstream or the module cache. It might be a good idea to vendor it by adding it to tools.go but that can happen in a followup PR.